corehunter-base/src/main/java/org/corehunter/data/simple/SimpleGenotypeVariantData.java

```diff
@@ -399,11 +401,27 @@ public final static void writeData(Path filePath,
         
         switch (format) {
             case DIPLOID:
-                writeDiploidData(filePath, genotypicData, type);
-                break ;
+                if (genotypicData instanceof GenotypeVariantData) {
```

Comment:
Actually, there is no real reason. I was following the readData signature.

Guy

> On 18 Apr 2016, at 14:24, Herman De Beukelaer <notifications@github.com> wrote:
> 
> In corehunter-base/src/main/java/org/corehunter/data/simple/SimpleGenotypeVariantData.java:
> 
> > @@ -399,11 +401,27 @@ public final static void writeData(Path filePath,
> >          
> >          switch (format) {
> >              case DIPLOID:
> > -                writeDiploidData(filePath, genotypicData, type);
> > -                break ;
> > +                if (genotypicData instanceof GenotypeVariantData) {
> Is there any reason to make the writeData method static? I think it would be more clean to make those regular instance methods so that you can override for biallelic data and avoid using instanceof.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>